x-pack/plugins/apm/public/components/app/Correlations/LatencyCorrelations.tsx

```diff
@@ -128,7 +128,7 @@ export function LatencyCorrelations() {
                 <EuiFormRow
                   fullWidth={true}
                   label="Field"
-                  helpText="Fields to analyse for significant terms"
+                  helpText="Fields to analyse for correlations"
```

Comment:
We didn't put in any i18n when doing the PoC. Should we start adding it here or open an issue to make sure it gets there for GA?

Reply:
I'll update https://github.com/elastic/kibana/issues/86477 to include the i18n.

Reply:
Thanks @ogupte. That's an oversigt on my part when I did the initial POC.